### ISSUE TITLE
History back - refetch 

### ISSUE THREAD

**Issue by peet86** [relay-tools/found-relay#157]
Hi! Is there a recommended way to trigger re-rendering / fetch when the user navigates back to the app with the browser's back button? I'm using FarceRouter. I need this functionality only for certain routes where I display data which can change on server-side while the user was "away" on another website / page.. 

 I could check for performance.navigation.type ===2 and do router.location.push({pathname: '/same', state:'reload'}) or something like this.. but some kind of high level implementation would be nicer. 

Could you please give me some directions / ideas how to implement this properly?

**Comment by taion** [comment]
I'm not sure what you mean – isn't the default behavior for the router to always re-do the data fetch, no matter how you get to the page?

**Comment by peet86** [comment]
Yes, it looks like another error or a small silent bug prevented this default behaviour in my app... or the alpha.12 release was buggy.. I have no idea.  I had to change a bunch of things here and there + I did a dependency update too (found-relay 0.3.1) in the last days and the problem is not there anymore, the routes works as expected.
